Zelda/Zelda.h

```diff
@@ -1,289 +0,0 @@
-#pragma once
-
-//---------------------Classes------------------------
-class HelperFunctions;
-class Item;
-class Treasure;
-class Weapon;
-class Monster;
-class Princess;
-class Player;
-class Room;
-class Castle;
-class Game;
-
-
-
-
-//---------------------Definitions------------------------
-
-
-
-
-
-//---------------------Helper Functions------------------------
-class HelperFunctions
-{
-private:
-
-public:
-
-	//Converts all the characters of Input to UpperCase
-	static void	charactersCase(char*);
-	static void color(int);
-};
-
-
-
-
-
-//------------------------Item----------------------------------
-class Item
-{
-protected:
-
-	char* name;
-
-public:
-
-	Item();
-	~Item();
-
-	inline char* getItemName();
-	inline virtual int getWorth() = 0;
-
-
-};
-
```

Comment:
Item is not a class that contains many variables. It may not be a good choice to do inheritance. 
Recommend to make both weapon and treasure to be independent class